plugin/manager.go

```diff
@@ -156,6 +157,10 @@ func (pm *Manager) HandleExitEvent(id string) error {
 	if restart {
 		pm.enable(p, c, true)
 	}
+
+	if err := mount.RecursiveUnmount(filepath.Join(pm.config.Root, id)); err != nil {
```

Comment:
For the enable case (line 158), is it okay to umount here?

Reply:
Good catch. Updated.